.dockerignore

```diff
@@ -0,0 +1,7 @@
+.git
+.github
+.gitignore
+.cache
```

Comment:
this probably needs to go into `.gitignore` too?

Reply:
done